.github/settings.yml

```diff
@@ -87,7 +87,7 @@ labels:
   - name: enhancement
     color: 02e10c
     description: New feature or request.
-  - name: enterpise docs
+  - name: enterprise docs
```

Comment:
I feel like I fixed this in another repo recently. Probably the typo exists in a few repos?

Reply:
Right, might be one worth running a grep for. I'll check.